app/src/main/res/layout/activity_main.xml

```diff
@@ -8,14 +8,25 @@
     tools:context=".activity.MainActivity">
 
     <TextView
-        android:id="@+id/text_view_main_app_title"
+        android:id="@+id/text_view_main_activity_app_title"
         style="@style/TextStyleAppTitle"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginTop="@dimen/main_activity_title_margin_xsmall"
+        android:layout_marginTop="@dimen/main_activity_margin_xsmall"
```

Comment:
I think that you can add this margin in the style

Reply:
done